docker-entrypoint.sh

```diff
@@ -1,9 +1,16 @@
-#!/bin/sh 
+#!/bin/bash 
 
 echo "Container starting up..." 
 set -e 
 
 cd ${BASEDIR}/${EDUMEET}/server
-node ${BASEDIR}/${EDUMEET}/server/server.js
+if [[ ! -e ${BASEDIR}/${EDUMEET}/server/dist/config/config.js ]] || [[ $(head -1 ${BASEDIR}/${EDUMEET}/server/config/config.js) != $(head -1 ${BASEDIR}/${EDUMEET}/server/dist/config/config.js) ]] 
+  then
+      echo "Rebuilding configuration..."
+      npm run build
+fi
+
```

Comment:
The new config mgmt is not yet merged to develop so let's hold back this changes until it will be merged.

Reply:
Actually, this is it is not in relations with new config mgmt, which, I hope, will resolve this "dirty workaround". 
Since .ts filetype is used, server app also has to be build, and if there is change in config.js file, it has to be rebuild again. This check is added to avoid unnecessary server app rebuilding every time docker image is started. There is comment in the docker-entrypoint.sh file which explains how this works. I really hope that there is better solution and this is only temporary workaround.